lib/Extractor/Solver.cpp

```diff
@@ -472,6 +472,94 @@ class BaseSolver : public Solver {
     return !IsSat;
   }
 
+  std::error_code testRange(const BlockPCs &BPCs,
+                            const std::vector<InstMapping> &PCs,
+                            Inst *LHS, llvm::APInt &C,
+                            llvm::APInt &X,
```

Comment:
can you find a better name for X and SynthesisX?

Reply:
How about naming X as "Result" and naming SynthesisX as "ReservedResult"?

Reply:
sounds good